webapp/client/src/components/FormFieldTaxNumber.test.js

```diff
@@ -0,0 +1,162 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { fireEvent } from "@testing-library/dom";
+import userEvent from "@testing-library/user-event";
+import FormFieldTaxNumber from "./FormFieldTaxNumber";
+
+describe("FormFieldTaxNumber is split", () => {
+  let props;
+  const exampleInput = "exampleInput";
+
+  beforeEach(() => {
+    props = {
+      fieldName: "fooName",
+      fieldId: "fooId",
+      label: {
+        text: "foo",
+        exampleInput: exampleInput,
+      },
+      errors: [],
+      values: [],
+      isSplit: true,
+    };
+    render(<FormFieldTaxNumber {...props} />);
+  });
+
+  it("Show three input fields", () => {
```

Comment:
Small stylistic note: the `describe` / `it` combo allows us to tell stories with tests suites, almost in complete sentences:

Describe: a boat
- it: should have sails
- it: should float

I like using the word "should" (although I realise some people have moved on from it) because of reasons well explained [here](https://dannorth.net/introducing-bdd/#an-expressive-test-name-is-helpful-when-a-test-fails):

> A more subtle aspect of the word should becomes apparent when compared with the more formal alternatives of will or shall. Should implicitly allows you to challenge the premise of the test: “Should it? Really?” This makes it easier to decide whether a test is failing due to a bug you have introduced or simply because your previous assumptions about the system’s behaviour are now incorrect.

Applied here, this would be:
```suggestion
  it("should show three input fields", () => {
```

or, if you don't like "should":
```suggestion
  it("shows three input fields", () => {
```

Very small change, but I find it makes a big difference, especially when applied consistently in a test suite.

Reply:
I 100% agree. Thanks for pointing these out.